app/js/components/CreateEscrowForm.jsx

```diff
@@ -103,4 +103,5 @@ CreateEscrowForm.propTypes = {
   result: PropTypes.object
 };
 
+export const CreateEscrowFormComponent = CreateEscrowForm;
```

Comment:
I guess you are doing that for the PropTypes.
Do you think it would be better if doing:
```
export class CreateEscrowForm {
}

export default withNamespaces()(CreateEscrowForm);
```

Then when you require, you have 2 form:
```
import CreateEscrowForm from './CreateEscrowForm';
```
and
```
import {CreateEscrowForm} from './CreateEscrowForm';
```
you could also 

```
import {CreateEscrowForm as CreateEscrowFormWithProptypes} from './CreateEscrowForm';
```

Just a suggestion, not sure it is really better

Reply:
I tried this option and couldn't make it work with Storybook. If you have some free time, please show me an example with a single component and I'll replicate the same behavior with the others